### DISCUSSION TITLE
Make org migrations endpoint accessible with fine-grained PAT

### DISCUSSION THREAD

**Discussion by Bisaloo** [community/community#38076] [Apps, API and Webhooks]
I'm currently using the [org migrations endpoint](https://docs.github.com/en/rest/migrations/orgs#about-the-organization-migrations-api) with a classic PAT and would like to switch to a new fine-grained PAT but it looks like it's not possible yet:

https://docs.github.com/en/rest/overview/permissions-required-for-fine-grained-personal-access-tokens

Could you add a specific permission for this on organization tokens?

**Comment by byrneh** [comment]
There is an official feedback discussion you can comments and track such questions on.   This existing reply may be of interest that mentions 

> Unfortunately Enterprise-level APIs and importer APIs are not supported yet for the fine-grained permissions platform that supports both GitHub Apps and fine-grained PATs.

It does mention its on the radar though
https://github.com/orgs/community/discussions/36441#discussioncomment-4019003

**Reply by ghostinhershell** [reply]
Thanks for linking to the official feedback discussion for the new fine-grained personal access tokens format @byrneh!
